WalletWasabi.Documentation/CodeCoverage.md

```diff
@@ -1,12 +1,35 @@
-Install `altcover` NuGet to the `WalletWasabi.Tests` project and Coverage Glutters extension to vscode.
+# Code coverage (How to)
+
+Wasabi wallet is build using dotnet core. Given there is no a cross-platform Profiling API like the one available on Windows, we 
+use [AltCover](https://github.com/SteveGilham/altcover) package for instrumenting the assemblies and recording the execution
+coverage.
+
+So, first of all we need to install the package in the `WalletWasabi.Tests` project as follow:
+
+```sh
+dotnet add WalletWasabi.Tests/WalletWasabi.Tests.csproj package AltCover
+```
+
+Next, we must build the solution as usual. After that, it is necessary to instrument the assemblies, run the tests and collect coverage:
```

Comment:
Do we need to build? Isn't it just simple `/p:AltCover=true /p:AltCoverLcovReport=lcov.info`?